suite/tests/api/detach_state.c

```diff
@@ -408,7 +408,94 @@ START_FILE
+        vmovups  [rsp +  3*SIMD_REG_SIZE], zmm3 @N@ \
+        vmovups  [rsp +  4*SIMD_REG_SIZE], zmm4 @N@ \
+        vmovups  [rsp +  5*SIMD_REG_SIZE], zmm5 @N@ \
+        vmovups  [rsp +  6*SIMD_REG_SIZE], zmm6 @N@ \
+        vmovups  [rsp +  7*SIMD_REG_SIZE], zmm7 @N@ \
+        vmovups  [rsp +  8*SIMD_REG_SIZE], zmm8 @N@ \
+        vmovups  [rsp +  9*SIMD_REG_SIZE], zmm9 @N@ \
+        vmovups  [rsp + 10*SIMD_REG_SIZE], zmm10 @N@ \
+        vmovups  [rsp + 11*SIMD_REG_SIZE], zmm11 @N@ \
+        vmovups  [rsp + 12*SIMD_REG_SIZE], zmm12 @N@ \
+        vmovups  [rsp + 13*SIMD_REG_SIZE], zmm13 @N@ \
+        vmovups  [rsp + 14*SIMD_REG_SIZE], zmm14 @N@ \
+        vmovups  [rsp + 15*SIMD_REG_SIZE], zmm15 @N@ \
+        vmovups  [rsp + 16*SIMD_REG_SIZE], zmm16 @N@ \
+        vmovups  [rsp + 17*SIMD_REG_SIZE], zmm17 @N@ \
+        vmovups  [rsp + 18*SIMD_REG_SIZE], zmm18 @N@ \
+        vmovups  [rsp + 19*SIMD_REG_SIZE], zmm19 @N@ \
+        vmovups  [rsp + 20*SIMD_REG_SIZE], zmm20 @N@ \
+        vmovups  [rsp + 21*SIMD_REG_SIZE], zmm21 @N@ \
+        vmovups  [rsp + 22*SIMD_REG_SIZE], zmm22 @N@ \
+        vmovups  [rsp + 23*SIMD_REG_SIZE], zmm23 @N@ \
+        vmovups  [rsp + 24*SIMD_REG_SIZE], zmm24 @N@ \
+        vmovups  [rsp + 25*SIMD_REG_SIZE], zmm25 @N@ \
+        vmovups  [rsp + 26*SIMD_REG_SIZE], zmm26 @N@ \
+        vmovups  [rsp + 27*SIMD_REG_SIZE], zmm27 @N@ \
+        vmovups  [rsp + 28*SIMD_REG_SIZE], zmm28 @N@ \
+        vmovups  [rsp + 29*SIMD_REG_SIZE], zmm29 @N@ \
+        vmovups  [rsp + 30*SIMD_REG_SIZE], zmm30 @N@ \
+        vmovups  [rsp + 31*SIMD_REG_SIZE], zmm31 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 0*OPMASK_REG_SIZE], k0 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 1*OPMASK_REG_SIZE], k1 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 2*OPMASK_REG_SIZE], k2 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 3*OPMASK_REG_SIZE], k3 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 4*OPMASK_REG_SIZE], k4 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 5*OPMASK_REG_SIZE], k5 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 6*OPMASK_REG_SIZE], k6 @N@ \
+        kmovw    [rsp + NUM_SIMD_REGS*SIMD_REG_SIZE + 7*OPMASK_REG_SIZE], k7 @N@ \
+        PUSH_GPRS
+#        elif defined(__AVX__)
+#            define PUSHALL                 \
+        lea      rsp, [rsp - NUM_SIMD_REGS*SIMD_REG_SIZE] @N@ \
+        vmovdqu  [rsp +  0*SIMD_REG_SIZE], ymm0 @N@ \
+        vmovdqu  [rsp +  1*SIMD_REG_SIZE], ymm1 @N@ \
+        vmovdqu  [rsp +  2*SIMD_REG_SIZE], ymm2 @N@ \
+        vmovdqu  [rsp +  3*SIMD_REG_SIZE], ymm3 @N@ \
+        vmovdqu  [rsp +  4*SIMD_REG_SIZE], ymm4 @N@ \
+        vmovdqu  [rsp +  5*SIMD_REG_SIZE], ymm5 @N@ \
+        vmovdqu  [rsp +  6*SIMD_REG_SIZE], ymm6 @N@ \
+        vmovdqu  [rsp +  7*SIMD_REG_SIZE], ymm7 @N@ \
+        vmovdqu  [rsp +  8*SIMD_REG_SIZE], ymm8 @N@ \
```

Comment:
Same as above.

Reply:
Same response: ifdef X64 is at line 374.

Reply:
Ok.